File: centerpark-back/src/main/java/model/Vehicule.java
package model;

public class Vehicule {
	private Integer id;
	private String model;

	public Vehicule(String model) {
		this.model = model;
	}
	
	public Vehicule(Integer id,String model) {
		this.id=id;
		this.model = model;
	}

	public String getModel() {
		return model;
	}

	public void setModel(String model) {
		this.model = model;
	}

	
	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	@Override
	public String toString() {
		return "Vehicule [id=" + id + ", model=" + model + "]";
	}

	
	
	
}

File: olivier/angular/formationAngular/src/app/notions/model/vehicule.ts
export class Vehicule {
  public constructor(
    private _id?: number | undefined,
    private _model?: string | undefined
  ) {}

  public get id(): number | undefined {
    return this._id;
  }

  public set id(value: number | undefined) {
    this._id = value;
  }

  public get model(): string | undefined {
    return this._model;
  }

  public set model(value: string | undefined) {
    this._model = value;
  }
}
